source/_integrations/google_assistant_sdk.markdown

```diff
@@ -45,30 +48,7 @@ You need to configure developer credentials to allow Home Assistant to access yo
 These credentials are the same as the ones for [Nest](/integrations/nest) or [Google Sheets](/integrations/google_sheets).
 If you have already set up credentials, you can do step 1 and then skip to step 13 on the below instructions.
```

Comment:
Can we maybe include this as well? Otherwise if the steps change we have to remind ourselves to manually update all the pages where we reference indiviual stesps

Reply:
Yes. But I would like to add more in separate PRs. I think maybe there are other elements we can streamline in these Google topics, but I don't want to do it all at once.